tennis-scheduler-web-app/src/components/TennisCourtAddChange.jsx

```diff
@@ -0,0 +1,162 @@
+                    setImage(response.data.image)
+                    setChangedType(response.data.surfaceType)
+                    setSurface(response.data.surfaceType)
+                }
+            )
+    }, [])
+
+    return (
+        <form onSubmit={onSubmit} className="addTennisCourt-form">
+            <div>
+                <label>Name</label>
+                <input type='text' className="addTennisCourt-input" id="name" name='name' placeholder="Add name of tennis court" value={tennisCourt.name} onChange={handleChange}></input>
+                <p className='errors'>{formErrors.name} </p>
+            </div>
+            <div>
+                <label>Description</label>
+                <textarea type='text' className="addTennisCourt-input" id="description" name='description' placeholder="Add description of tennis court" value={tennisCourt.description} onChange={handleChange}></textarea>
+                <p className='errors'>{formErrors.description} </p>
+            </div>
+            <div>
+                <label>Image</label>
+                <input type="file" className="form-control" onChange={setImages} />
+            </div>
+            {
+                check ? <img className='picture-preview' src={require('../images/' + image)} /> : ""
+            }
+            <select className="form-select select-option" value={changedType} onChange={handleChangeSurfaceType}>
+                <option value="GRASS" onSelect={handleChangeSurfaceType}>GRASS</option>
+                <option value="CLAY" onSelect={handleChangeSurfaceType}>CLAY</option>
+                <option value="HARD" onSelect={handleChangeSurfaceType}>HARD</option>
+            </select>
+            <p className='errors'>{formErrors.surfaceType} </p>
+            <div>
+                <label>Country</label>
+                <input type='text' placeholder="Add country" className="addTennisCourt-input" id="country" name='country' value={address.country} onChange={handleChangeAddress}></input>
+                <p className='errors'>{formErrors.country} </p>
+            </div>
+            <div>
+                <label>City</label>
+                <input type='text' placeholder="Add city" id="city" name='city' className="addTennisCourt-input" value={address.city} onChange={handleChangeAddress} ></input>
+                <p className='errors'>{formErrors.city} </p>
+            </div>
+            <div>
+                <label>Street</label>
+                <input type='text' placeholder="Add street" id="street" name='street' className="addTennisCourt-input" value={address.street} onChange={handleChangeAddress}></input>
+                <p className='errors'>{formErrors.street} </p>
+            </div>
+            <div>
+                <label>Number</label>
+                <input type='number' placeholder="Add number" id="number" name='number' className="addTennisCourt-input" value={address.number} onChange={handleChangeAddress}></input>
```

Comment:
You don't need to change this now. But it is always preferable to use one type of quotation marks only and not mix it. If you use '' in set state, do not use "".